package.json

```diff
@@ -128,7 +128,7 @@
     "sequelize-cli": "^2.3.1",
     "speakingurl": "^9.0.0",
     "sqlite3": "^3.1.8",
-    "steem": "github:steemit/steem-js#72a36c835a74f446da47e36feea5d5f1f4dde55b",
+    "@steemit/steem-js": "^0.6.5",
```

Comment:
peg the exact version number. don't caret this

Reply:
My intention was to have the lockfile be the source of truth for exact versions but we could do both

Reply:
I want it to be a conscious decision for us to change the version number. could be excessively paranoid however. @sneak your thoughts?

Reply:
exact version AND lockfile checksum.